src/player/player.controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import { PlayerService } from './player.service';
+
+@Controller('players')
+export class PlayerController {
+  constructor(private readonly playerService: PlayerService) {}
+
+  @Get(':id')
+  async findOne(@Param('id') id: number) {
```

Comment:
Id parameter needs to be parsed into integer and exception needs to be handled
[Nest JS pipes](https://docs.nestjs.com/pipes)